numpy/ma/core.py

```diff
@@ -5769,6 +5775,14 @@ def min(self, axis=None, out=None, fill_value=None, keepdims=np._NoValue):
         ma.minimum_fill_value
             Returns the minimum filling value for a given datatype.
 
+        Examples
+        --------
+
+        >>>  x = np.ma.array([1, 3, 4, 6], mask=[0, 0, 0, 1])
```

Comment:
```suggestion
        >>> x = np.ma.array([1, 3, 4, 6], mask=[0, 0, 0, 1])
```
There should only be a single space here, this also trips the test suite.